lib/react_on_rails/version_checker.rb

```
@@ -39,30 +34,29 @@ def common_error_msg
       Rails.logger.warn(msg)
     end
 
     def gem_version
       ReactOnRails::VERSION
     end
 
-    def gem_major_minor_patch_version
-      match = gem_version.match(MAJOR_MINOR_PATCH_VERSION_REGEX)
-      [match[1], match[2], match[3]]
+    def gem_version_parts
+      gem_version.match(VERSION_PARTS_REGEX)&.captures.compact
```

RuboCop (1.75.5)

[warning] Do not chain ordinary method call after safe navigation operator.

(Lint/SafeNavigationChain)